doc/ch-fromhost_desc.rst

```diff
@@ -195,6 +240,10 @@ There are two alternate approaches for nVidia GPU libraries:
 Further, while these alternate approaches would simplify or eliminate this
 script for nVidia GPUs, they would not solve the problem for other situations.
 
+At the time of this writing, a Cray Slingshot optimized provider is not
+available. We are working with HPE to get this feature added sooner, rather
+than later, however, we may need to revert to more complicated injection
+techniques for future Cray systems with Slingshot.
```

Comment:
Is this an argument to keep the old Cray options?

Reply:
The intention was to imply that we may need to visit _overwriting the container libfabric_ with the host's. A last resort would be to revisit overwriting the container MPI.

Reply:
OK. Maybe clarify that explanation?

In general, I think we can trust our users with the technical explanation in the docs. If it's too messy, it could be segregated into a “messy details here” type section.